applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -258,37 +206,31 @@ static void stream_configured_cb(struct bt_audio_stream *stream,
 	int ret;
 	uint8_t channel_index;
 
-	ret = channel_index_get(stream->conn, NULL, &channel_index);
+	ret = channel_index_get(stream->conn, &channel_index);
 	if (ret) {
-		LOG_ERR("Stream not found");
+		LOG_ERR("Channel index not found");
 		return;
 	}
 
 	if (stream->ep->dir == BT_AUDIO_DIR_SINK) {
-		headsets[channel_index].sink_stream = stream;
```

Comment:
I may be blind, but where are the sink_stream and the source_stream now set?

Reply:
I've seen that they are no longer pointers, and I assume set through the stream_config